app/api/v1/endpoints/media.py

```
@@ -437,68 +437,81 @@ async def get_media_signed(
-
-                if response.status_code in (401, 403, 404, 416):
-                    await _close_httpx_stream(response)
-                    # Map proxied status codes
-                    if response.status_code == 404:
-                         raise HTTPException(status_code=404, detail="Media not found")
-                    raise HTTPException(status_code=response.status_code, detail="Provider error")
+                # Session closed, but streaming response is still open
+            except ValueError as e:
+                raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e)) from None
+            except Exception as e:
+                error_logger.exception(f"Proxy original failed: {e}")
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)